src/fairchem/core/common/relaxation/optimizers/lbfgs_torch.py

```diff
@@ -106,31 +89,32 @@ def run(self, fmax, steps):
                 ase.io.Trajectory(self.traj_dir / f"{name}.traj_tmp", mode="w")
                 for name in self.traj_names
             ]
+            self.write()
```

Comment:
Do we need this here? https://github.com/FAIR-Chem/fairchem/pull/718/files#diff-70fccaaa1dc6b26e2f89e34ac64bd162d1c7f26b725f861c3ce58b1c33924ffdR104 is called before your first step so you are already writing the first frame there.

Reply:
good catch!